tests/screenshotter/config/tabs-v2.json

```diff
@@ -0,0 +1 @@
"2de424de","settings":{"background_background":"classic","background_color":"#F4CC02"},"elements":[{"id":"45843a5b","settings":{"title":"Content -&gt; Style -&gt; Border Width - 10","header_size":"h5","__globals__":{"title_color":"globals\/colors?id=secondary"}},"elements":[],"isInner":false,"widgetType":"heading","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"734bff66","settings":[],"elements":[{"id":"50dfd79b","settings":{"tabs":[{"tab_title":"Tab #1","_id":"3cda4e8"},{"tab_title":"Tab #2","_id":"a7faae4"}],"border_width":{"unit":"px","size":10,"sizes":[]}},"elements":[{"id":"1d6f40f3","settings":{"_title":"Tab #1"},"elements":[{"id":"62037790","settings":{"tabs":[{"tab_title":"Tab #1","_id":"02d989c"},{"tab_title":"Tab #2","_id":"0dbace0"}]},"elements":[{"id":"4ff67495","settings":{"_title":"Tab #1"},"elements":[{"id":"3722028e","settings":{"title":"CONTENT"},"elements":[],"isInner":false,"widgetType":"heading","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"3c881010","settings":{"_title":"Tab #2"},"elements":[],"isInner":false,"elType":"container"}],"isInner":false,"widgetType":"tabs-v2","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"2fdae62b","settings":{"_title":"Tab #2"},"elements":[],"isInner":false,"elType":"container"}],"isInner":false,"widgetType":"tabs-v2","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"1091ac61","settings":{"background_background":"classic","background_color":"#F4CC02"},"elements":[{"id":"7d4ddc37","settings":{"title":"Content -&gt; Style -&gt; Border Width - 10 &amp;&amp; Nested Tab -&gt; Content -&gt; Style -&gt; Border Width - 10<br>","header_size":"h5","__globals__":{"title_color":"globals\/colors?id=secondary"}},"elements":[],"isInner":false,"widgetType":"heading","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"5cfc0b34","settings":[],"elements":[{"id":"238336c0","settings":{"tabs":[{"tab_title":"Tab #1","_id":"3cda4e8"},{"tab_title":"Tab #2","_id":"a7faae4"}],"border_width":{"unit":"px","size":10,"sizes":[]}},"elements":[{"id":"256033e5","settings":{"_title":"Tab #1"},"elements":[{"id":"413c653e","settings":{"tabs":[{"tab_title":"Tab #1","_id":"02d989c"},{"tab_title":"Tab #2","_id":"0dbace0"}],"border_width":{"unit":"px","size":10,"sizes":[]}},"elements":[{"id":"6832a7a6","settings":{"_title":"Tab #1"},"elements":[{"id":"1cb43190","settings":{"title":"CONTENT"},"elements":[],"isInner":false,"widgetType":"heading","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"60f376d0","settings":{"_title":"Tab #2"},"elements":[],"isInner":false,"elType":"container"}],"isInner":false,"widgetType":"tabs-v2","elType":"widget"}],"isInner":false,"elType":"container"},{"id":"5ba57982","settings":{"_title":"Tab #2"},"elements":[],"isInner":false,"elType":"container"}],"isInner":false,"widgetType":"tabs-v2","elType":"widget"}],"isInner":false,"elType":"container"}],"page_settings":[],"version":"0.4","title":"Tabs-V2","type":"page"}
```

Comment:
Please reduce the page length by combine the same control test to columns in the same row.

Reply:
Handled.